### PR TITLE
Some minor things and refactorings.

### PR DESCRIPTION
**Pull request by vinjana**
Fix printidlessruntimeconfig command (required job manager).

### CODE REVIEW

File: RoddyCore/src/de/dkfz/roddy/client/RoddyStartupModeScopes.java [hunk 1]
```diff
@@ -13,36 +13,42 @@
 public enum RoddyStartupModeScopes {
 
     /**
-     * Use this scope to leave options for command line processing.
-     * They will not be used in the Roddy binary!
+     * Use this scope to ignore options for command line processing. They will not be used in the Roddy binary!
+     * CLI scope is used for startup modes in which the Roddy binary is not active, except for
```
Comment (dankwart-de) [inline RoddyCore/src/de/dkfz/roddy/client/RoddyStartupModeScopes.java]: Except for? Something's missing

File: RoddyCore/src/de/dkfz/roddy/client/RoddyStartupModeScopes.java [hunk 2]
```diff
@@ -13,36 +13,42 @@
...
      * Instead, they need a script file and are fully handled in the roddy.sh
      * This is useful, if things can be handled in an easier way with scripts,
      * like i.e. file operations
+     *
+     * no execution service, no execution service, no filesystem access provider
```
Comment (dankwart-de) [inline RoddyCore/src/de/dkfz/roddy/client/RoddyStartupModeScopes.java]: 2x no exec serv?

File: dist/bin/current/helperScripts/resolveAppConfig.sh [hunk 3]
```diff
@@ -51,7 +51,12 @@ function setRoddyBinaryVariables() {
   fi
   local numberOfDots=$(grep -o "[.]" <<< "$useRoddyVersion" | wc -l)    # Try find out, if we got an api level input like 2.2 or 2.3
   if [[ $numberOfDots == 1 ]]; then
-    useRoddyVersion=$(cd $RODDY_DIRECTORY/dist/bin; ls -d $useRoddyVersion* | sort -V | tail -n 1)
+    useRoddyVersionDir=$(cd $RODDY_DIRECTORY/dist/bin; find -maxdepth 1 -name "$useRoddyVersion*")
```
Comment (dankwart-de) [inline dist/bin/current/helperScripts/resolveAppConfig.sh]: I guess the * is to get the latest entry when using the short syntax? So e.g. 2.3 => 2.3.112?

But how about directory order returned by find?

Comment (vinjana) [reply dist/bin/current/helperScripts/resolveAppConfig.sh]: The directory order is fixed to version order below.

**Pull request merged**